README.rst

```diff
@@ -175,18 +175,19 @@ PDAL and Python:
     print(pipeline.execute())  # 387 points
     clamped = pipeline.arrays[0]
 
-    # Write our intensity data to an LAS file
-    pipeline = pdal.Writer.las(
-        filename="clamped2.las",
-        offset_x="auto",
-        offset_y="auto",
-        offset_z="auto",
-        scale_x=0.01,
-        scale_y=0.01,
-        scale_z=0.01,
-    ).pipeline(clamped)
+    # Write our intensity data to a TileDB array. TileDB uses Hilbert ordering,
```

Comment:
# Write our intensity data to a TileDB array. It is recommended to use Hilbert ordering by default with geospatial point cloud data.